src/styles/styles.css

```diff
@@ -0,0 +1,4 @@
+@import "./card.css";
+@import "./reset.css";
+@import "./stars.css";
+@import "./variables.css";
```

Comment:
You need to import files in order. Some base styles and variable which can effort on your project should be first in importing